testsuite/tests/statmemprof/lists_in_minor.opt.reference

```diff
@@ -0,0 +1,13 @@
+check_distrib 10 1000000 0.010000
+check_distrib 1000000 10 0.000010
+check_distrib 1000000 10 0.000100
+check_distrib 1000000 10 0.001000
+check_distrib 1000000 10 0.010000
+check_distrib 100000 10 0.100000
+check_distrib 100000 10 0.900000
+check_callstack
+Raised by primitive operation at file "lists_in_minor.ml", line 15, characters 25-34
```

Comment:
Is there a reason why the two reference files are different?

I just figured out that the bytecode reference file is bogus, since the allocation point does not appear there. This needs to be fixed.

Reply:
I'll have a look at this in a separate PR, I think it's a bytecode debug info issue, unrelated to this.

Reply:
FWIW, I have identified the problem(s) and I am currently working on a fix. More to come soon.

Reply:
I've just pushed some code that generates separate backtraces for individual comballoc'd blocks, which seems to suffer the same issue (see `comballoc.byte.reference` and `comballoc.opt.reference`)